www/js/diary/list.js

```diff
@@ -37,15 +40,6 @@ angular.module('emission.main.diary.list',['ui-leaflet',
       readAndUpdateForDay($rootScope.barDetailDate);
       $rootScope.barDetail = false;
     };
-    if($rootScope.displayingIncident == true) {
-      if (angular.isDefined(Timeline.data.currDay)) {
-          // page was already loaded, reload it automatically
-          readAndUpdateForDay(Timeline.data.currDay);
```

Comment:
I will merge this for now, but note that you may want to restore it after testing since the trip end surveys were removed during the pilot in `cci-berkeley`

Reply:
I will revert this one back to what it was. Thank you.